CMakeLists.txt

```diff
@@ -25,6 +25,15 @@ include(os_release_info.cmake)
 
 get_os_release_info(os_name os_version)
 
+if (CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT OR CMAKE_INSTALL_PREFIX STREQUAL "")
+    set(CMAKE_INSTALL_PREFIX "/usr" CACHE PATH "..." FORCE)
```

Comment:
I think it should be '/usr/local' as default. Usual practice is to step out from /usr and avoid rewriting system defaults (you should assume that media-driver is already installed from the system packages).

Reply:
good point! Hi Eric, please reconsider this. I think we actually don't need to do anything, just depending on GNUInstallDirs default setting on a distribution should be fine. If we really need a package which could overwrite system version, just passing CMAKE_INSTALL_PREFIX in cmake command line should be fine.

Reply:
Ok, i will edit this to just deal with edge case mentioned in #198, ie. CMAKE_INSTALL_PREFIX=